validator/tests/test_service.py

```diff
@@ -6,7 +6,7 @@
 
 @pytest.mark.parametrize(['data', 'errormsg'], [
     (None, 'JSON payload missing'),
-    ('[]', 'Payload does not contain a "data" field'),
+    ('42', 'Payload must be a JSON object'),
```

Comment:
Why did you remove the `'[]'` test case?

Reply:
It's not applicable anymore, since we're now doing an `isinstance` check.